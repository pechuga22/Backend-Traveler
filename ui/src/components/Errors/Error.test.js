import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { render } from '@testing-library/react'

import Error from './Error'

describe('./components/Error/Error.js', () => {
  describe('.render()', () => {
    it('renders text Error', () => {
      const { getByText } = render(<Error />)
      const loadingText = getByText(/fetching/i)
    }) 

    it('should render Error component', () => {
      const component = shallow(<Error />)
      expect(toJson(component)).toMatchSnapshot()
    })
  })
})
