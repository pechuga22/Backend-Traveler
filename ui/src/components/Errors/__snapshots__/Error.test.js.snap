// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./components/Error/Error.js .render() should render Error component 1`] = `
<Message
  icon={true}
>
  <Icon
    as="i"
    loading={true}
    name="circle notched"
  />
  <MessageContent>
    <MessageHeader>
      Just one second
    </MessageHeader>
    We are fetching that content for you.
  </MessageContent>
</Message>
`;
